### PR TITLE
Move int64s to top of struct to resolve alignment issue on ARM

### PR DESCRIPTION
**Pull request by ethanadams**
What: 
Move int64s used in atomic operations to top of struct
Why:
Resolves alignment issue on ARM (https://github.com/golang/go/issues/23345)

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: storagenode/storagenodedb/infodb.go [hunk 1]
```diff
@@ -44,8 +44,8 @@ func newInfo(path string) (*InfoDB, error) {
 	dbutil.Configure(db, mon)
 
 	infoDb := &InfoDB{db: db}
-	infoDb.pieceinfo = pieceinfo{infoDb, spaceUsed{sync.Once{}, 0}}
-	infoDb.bandwidthdb = bandwidthdb{infoDb, bandwidthUsed{sync.RWMutex{}, time.Time{}, 0}}
+	infoDb.pieceinfo = pieceinfo{infoDb, spaceUsed{0, sync.Once{}}}
```
Comment (jtolds) [inline storagenode/storagenodedb/infodb.go]: okay for this PR buuuut


definitely a strong preference to not create structs with unnamed, positional fields. always always use named fields in struct creation.

File: storagenode/storagenodedb/pieceinfo.go [hunk 2]
```diff
@@ -24,8 +24,8 @@ type pieceinfo struct {
 }
 
 type spaceUsed struct {
-	once sync.Once
 	used int64
+	once sync.Once
```
Comment (egonelbre) [inline storagenode/storagenodedb/pieceinfo.go]: Add a comment why it's in a different order.

File: storagenode/storagenodedb/bandwidthdb.go [hunk 3]
```diff
@@ -22,9 +22,9 @@ type bandwidthdb struct {
 }
 
 type bandwidthUsed struct {
+	used      int64
```
Comment (egonelbre) [inline storagenode/storagenodedb/bandwidthdb.go]: Add a comment why it's in a different order.

File: storagenode/storagenodedb/infodb.go (hunk 1 above)
Comment (ethanadams) [reply storagenode/storagenodedb/infodb.go]: fixed

File: storagenode/storagenodedb/pieceinfo.go (hunk 2 above)
Comment (ethanadams) [reply storagenode/storagenodedb/pieceinfo.go]: done

File: storagenode/storagenodedb/bandwidthdb.go (hunk 3 above)
Comment (ethanadams) [reply storagenode/storagenodedb/bandwidthdb.go]: done

**Comment by wthorp** [comment]
> Looks fine, but what do you mean by an alignment issue?

The's a known issue with certain 'atomic' methods on 64-bit datatypes on certain ARM processers.
https://golang.org/pkg/sync/atomic/#pkg-note-BUG

**Pull request merged**